src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -243,6 +243,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
     backendCall.instantiateCommand match {
       case Success(command) => runWithJes(backendCall, command, jesInputs, jesOutputs, runIdForResumption)
+      case Failure(ex: SocketTimeoutException) => throw ex // probably a GCS transient issue, throwing will cause it to be retried
```

Comment:
Probably more of question for @mcovarr, though I too find the different between throw and message passing a bit confusing but perhaps there's a good reason that I'm not remembering